test/lib/routes/index.test.js

```diff
@@ -191,13 +193,18 @@ describe('Application routes', function () {
     });
 
     it('sends the payload to the feedsme service after a successful build', function (next) {
-
+      const feedStub = sinon.stub(app.feedsme, 'change').yieldsAsync(null, null);
+      const pay = getPayload(payload);
+      pay.__published = true;
       nockFeedme();
 
       fs.createReadStream(payload).pipe(createRequest('post', 'build'))
         .on('data', validateMessages)
-        .on('end', next)
-        .on('error', next);
+        .on('error', next)
+        .on('end', () => {
+          assume(feedStub).is.calledWithMatch('dev', sinon.match.has('data', sinon.match.has('__published', true)), sinon.match.func);
```

Comment:
```suggestion
          assume(feedStub).is.calledWithMatch('dev', sinon.match.hasNested('data.__published', true), sinon.match.func);
```
Nit, but `hasNested` is a little more succinct.